example/src/views/login/screen3.js

```diff
@@ -1,22 +1,280 @@
+    this.signUp = this.signUp.bind(this);
+  }
+
+  async componentDidMount() {
+    await Font.loadAsync({
+      'georgia': require('../../../assets/fonts/Georgia.ttf'),
+      'regular': require('../../../assets/fonts/Montserrat-Regular.ttf'),
+      'light': require('../../../assets/fonts/Montserrat-Light.ttf'),
+    });
+
+    this.setState({ fontLoaded: true });
+  }
+
+  selectCategory(selectedCategory) {
+    LayoutAnimation.easeInEaseOut();
+    this.setState({
+      selectedCategory,
+      isLoading: false,
+    });
+  }
+
+  validateEmail(email) {
+    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+    return re.test(email);
+  }
+
+  login() {
+    const {
+      email,
+      password,
+    } = this.state;
+    this.setState({ isLoading: true });
+    // Simulate an API call
+    setTimeout(() => {
+      LayoutAnimation.easeInEaseOut();
+      this.setState({
+        isLoading: false,
+        isEmailValid: this.validateEmail(email),
+        isPasswordValid: password.length >= 8
+      });
+    }, 1500);
+  }
+
+  signUp() {
+    const {
+      email,
+      password,
+      passwordConfirmation,
+    } = this.state;
+    this.setState({ isLoading: true });
+    // Simulate an API call
+    setTimeout(() => {
+      LayoutAnimation.easeInEaseOut();
+      this.setState({
+        isLoading: false,
+        isEmailValid: this.validateEmail(email),
+        isPasswordValid: password.length >= 8,
+        isConfirmationValid: password == passwordConfirmation,
+      });
+    }, 1500);
+  }
+
   render() {
+    const {
+      selectedCategory,
+      isLoading,
+      isEmailValid,
+      isPasswordValid,
+      isConfirmationValid,
+      email,
+      password,
+      passwordConfirmation,
+    } = this.state;
+    const isLoginPage = selectedCategory === 0;
+    const isSignUpPage = selectedCategory === 1;
     return (
       <View style={styles.container}>
         <Image
           source={BG_IMAGE}
           style={styles.bgImage}
         >
-          <Text style={styles.loginText}>
-            Login Screen 3
-          </Text>
+          {this.state.fontLoaded ?
+            <View>
+              <KeyboardAvoidingView contentContainerStyle={styles.loginContainer} behavior='position'>
+                <View style={styles.titleContainer}>
+                  <View style={{flexDirection: 'row'}}>
+                    <Text style={styles.titleText}>BEAUX</Text>
+                  </View>
+                  <View style={{marginTop: -10, marginLeft: 10}}>
+                    <Text style={styles.titleText}>VOYAGES</Text>
+                  </View>
+                </View>
+                <View style={{flexDirection: 'row'}}>
+                  <Button
+                    disabled={isLoading}
+                    buttonStyle={{backgroundColor: 'transparent'}}
+                    underlayColor={'transparent'}
```

Comment:
I have added a `clear` prop on the Button component. So you can ideally replace
```
buttonStyle={{backgroundColor: 'transparent'}}
underlayColor={'transparent'}
``` 
with the `clear` prop.